code/script.js

```diff
@@ -3,17 +3,155 @@
+    `);
+    typeOf = ["Quattro Formaggi", "Pepperoni", "Margherita", "Calzone"];
+    break;
+  case "Pasta":
+    menuFoodSelected = prompt(`Please select what you would like by entering the number:
+
+    1- Carbonara
+    2 - Bolognaise 
+    3 - Cacio e pepe
+    4 - Spaghetti allo Scoglio
+    `);
+    typeOf = ["Carbonara", "Bolognaise", " Cacio e pepe", "Spaghetti allo Scoglio"];
+    break;//obs, don't forget square baccets for the arry
+  case "Salad":
+    menuFoodSelected = prompt(`Please select what you would like by entering the number:
+
+    1 - Green Godess
+    2 - Rainbow
+    3 - Greek
+    4 - Fruit
+    `);
+    typeOf = ["Green Godess", "Rainbow", "Greek", "Fruit"];
+    break;
+  case "Dessert":
+    menuFoodSelected = prompt(`Please select what you would like by entering the number:
+
+    1 - Tiramisu
+    2 - Gelato
+    3 - Chocolate Fondant
+    4 - Surprise dish: Yummy in my tummy
+    `);
+    typeOf = ["Tiramisu", "Gelato", "Chocolate Fondant", "Surprise dish; Yummy in my tummy"];
+    break;
+}
+
+if (menuFoodSelected === "1") {
+  alert(
+    `Great Choice, You have chosen: ${typeOf[0]} as your ${selectedOption}!`);
+} else if (menuFoodSelected === "2") {
+  alert(
+    `Great Choice, You have chosen: ${typeOf[1]} as your ${selectedOption}!`);
+} else if (menuFoodSelected === "3") {
+  alert(
+    `Great Choice, You have chosen: ${typeOf[2]} as your ${selectedOption}!`);
+}
+else if (menuFoodSelected === "4") {
+  alert(
+    `Great Choice, You have chosen: ${typeOf[3]} as your ${selectedOption}!`);
+}
+else {
+  alert(`Invalid choise, please select a number between 1-4 to make an order`)
+}
+//Should I have used the parseInt here as well to simplyfy the code? Would that make it easier to specify texts for each of the foodtypes?
+
 // Step 4 - Age
 // Your code goes here
+// I really wanted to figure out how to get a message to display  the choises made and the associated cost together with the confirmation promt in one message. Do I have to change this to an alert and a promt seperated from eachother to meet the requrements?
+
+let order = "";
+
+let age = prompt("Please enter your age to choose if the dish is meant for an adult or a child.")
+if (age >= 18) {
+  order = prompt(`You have chosen an adult sized ${typeOf[parseInt(menuFoodSelected) - 1]} ${selectedOption}. That will be €20.
+
+  Enter Y for "Yes" to confirm.
+  Enter N for "No" to decline`);
+}
+else if (age <= 18) {
+  order = prompt(`You have chosen an child sized ${typeOf[parseInt(menuFoodSelected) - 1]} ${selectedOption}. That will be €15.
+  
+  Enter Y for "Yes" to confirm.
+  Enter N for "No" to decline`);
+}
+//I spent a lot of time trying to figgure out the right way to write the code so that the typeOf array chosen earlier would be the same here as well. I kept trying with ${typeOf[0]} ${typeOf[0, 1 , 2 , 3]} and so on. I got help on slack and during the demos! Not sure if I copletly understand the "parseInt" yet, but for now it works. 
```

Comment:
This explanation helped me with this:  "menuFoodSelected" is a string, and it could be "1" to "3". So you should convert to a number, in order to use it as an index number. Then you need to subtract 1 (-1), as "typeOf" is an array and the index starts from 0.